Formula/sshguard.rb

```diff
@@ -70,6 +75,6 @@ def plist; <<-EOS.undent
   end
 
   test do
-    assert_match version.to_s, shell_output("#{sbin}/sshguard -v 2>&1", 78)
+    assert_match "SSHGuard #{version.to_s}", shell_output("#{sbin}/sshguard -v 2>&1")
```

Comment:
`W: 78: 38: Redundant use of Object#to_s in interpolation.`

Just use `"SSHGuard #{version}"` and I believe we are good to go!